examples/data-sources/cloudinit_config/data-source.tf

```diff
@@ -0,0 +1,18 @@
+data "cloudinit_config" "foobar" {
+  gzip          = false
+  base64_encode = false
+
+  part {
+    content = "foo"
+
+    content_type = "text/x-shellscript"
+    filename     = "foo.sh"
```

Comment:
Nit: While we're here, it may be worth updating these examples to be more realistic. As an unfamiliar practitioner, it may not be clear what "foo" is here or why there is "foo" and "foo.sh".

Reply:
good call. I'll update both data source + resource with a better example